src/Symfony/Bridge/Doctrine/Tests/Serializer/Entity/Entity.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
This should be moved to a Fixture/s directory, like done in other components. Thanks